### PR TITLE
User Identification via HubSpot Integration

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Enabled HubSpot
Added the user identification script according to https://developers.hubspot.com/docs/api/events/tracking-code#identify-a-visitor

## Related Issue(s)

closes [#3392](https://github.com/FlowFuse/flowfuse/issues/3392)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [x] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by Yndira-FlowForge** [comment]
Hi @knolleary 
I've added a script to enable HubSpot's survey and chat features, as well as to identify visitors. To identify the visitors, I'm fetching the user's email and sending it to HubSpot. I wanted to check with you if this is the best approach. Is there a need to fetch the user's email like this, or is it already available through another method?

File: etc/flowforge.yml
Comment (knolleary) [inline etc/flowforge.yml:1]: We don't need to add this to the default yml file - particular our own tracking code.

File: forge/routes/ui/index.js
Comment (knolleary) [inline forge/routes/ui/index.js:1]: I think we should be doing this in the Vue code of the application - as that will already have identified the current logged-in user and have the information available and avoids the need for this duplicate API call.

That will need a bit more work as we don't currently pass the hubspot config over to the Vue code... but I have an idea. I'll pick this up if you'd like.

**Review by knolleary — CHANGES_REQUESTED** [review]

File: forge/routes/ui/index.js
Comment (Yndira-FlowForge) [reply forge/routes/ui/index.js:1]: Thank you @knolleary

> I'll pick this up if you'd like.

Yes, please, I haven't worked with Vue before 🙃

**Comment by knolleary** [comment]
Closing this PR as it will be superseded by #3411

**Pull request closed without merging**